modules/pipeline/services/pipelinesvc/detail.go

```diff
@@ -476,3 +477,10 @@ func findRunningStageID(p spec.Pipeline, tasks []spec.PipelineTask) uint64 {
 	}
 	return runningStageID
 }
+
+// isSuccessTaskEvents if events contain Successfully assigned like
+// "Successfully assigned project-387-dev/pipeline-3feb156fc4-65d7476d75-7m6qm to node-010000006200"
+// means scheduler successfully execute task
+func isSuccessTaskEvents(events string) bool {
+	return strings.Contains(events, "Successfully assigned")
```

Comment:
all events don't contain `Successfully assigned` are failed?

Reply:
if task it`s not success and task don`t asigned, should reveal the events